package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sobject",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Thanks for remembering to bump this. I'll merge this and then run the job to publish the updated package to NPM.